get_all_manifests.sh

```diff
@@ -18,6 +18,7 @@ declare -A COMPONENT_MANIFESTS=(
     ["kf-notebook-controller"]="opendatahub-io:kubeflow:v1.7-branch:components/notebook-controller/config:odh-notebook-controller/kf-notebook-controller"
     ["odh-notebook-controller"]="opendatahub-io:kubeflow:v1.7-branch:components/odh-notebook-controller/config:odh-notebook-controller/odh-notebook-controller"
     ["notebooks"]="opendatahub-io:notebooks:main:manifests:notebooks"
+    ["trustyai"]="trustyai-explainability:trustyai-service-operator:main:config:trustyai-service-operator"
```

Comment:
IIUC, this would fetch the manifests from `main` (TrustyAI's dev branch) which might not be aligned with the released ones. 

TrustyAI uses release branches (latest https://github.com/trustyai-explainability/trustyai-service-operator/tree/release/1.11.1), so this would be

`["trustyai"]="trustyai-explainability:trustyai-service-operator:release/1.11.1:config:trustyai-service-operator"`

But we can change upstream to better suit the odh-operator process.